### PR TITLE
Rename all gradle properties to Mavericks

### PR DESCRIPTION
**Pull request by gpeal**

### CODE REVIEW

File: mvrx-rxjava2/gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-POM_NAME=MvRx
-POM_ARTIFACT_ID=mvrx-rxjava2
+POM_NAME=Mavericks
+POM_ARTIFACT_ID=mavericks-rxjava2
```
Comment (elihart) [inline mvrx-rxjava2/gradle.properties:2]: should this pr include updates to the docs that reference these artifacts as well?

**Review by elihart — APPROVED** [review]

**Comment by gpeal** [comment]
@elihart Good call! Updated the docs. It's probably worth doing _one_ more to update the module names.

**Comment by gpeal** [comment]
@elihart Can you take another look at this?

**Review by elihart — APPROVED** [review]
@gpeal looks great, thanks! Still planning to time merging this with when we do the repo rename?

**Comment by gpeal** [comment]
@elihart I don't think this needs to wait for the repo rename. Do you?

**Comment by elihart** [comment]
The readme details and docs links won't be right until we do, and there may be some confusion in people on pre-2.0 versions that look at it

**Pull request merged**